scene/3d/navigation_region_3d.h

```diff
@@ -38,11 +38,11 @@ class NavigationRegion3D : public Node3D {
 	GDCLASS(NavigationRegion3D, Node3D);
 
 	bool enabled = true;
-	RID region;
-	Ref<NavigationMesh> navmesh;
-
+	RID region = RID();
```

Comment:
AFAIK RIDs are default initialized to an empty/invalid RID so this assignment is redundant (same in 2D).

Reply:
Fixed.